bindings/2.200/GeometryDash.bro

```diff
@@ -133,6 +133,29 @@ class ButtonSprite : cocos2d::CCSprite {
 	~ButtonSprite();
 }
 
+class CreatorLayer : cocos2d::CCLayer {
```

Comment:
missing the link(android) attr, but you should instead of just copy off GeometryDash-extras.bro (and delete from there once you paste it here!) so that it also has all other methods